frontend/src/components/Arrow/Arrow.js

```diff
@@ -3,62 +3,44 @@ import cx from 'classnames';
 import Icon from '../../components/Icons';
 import styles from './styles.module.scss';
 
-class Arrow extends React.PureComponent {
-  renderIcon() {
-    if (this.props.left) {
-      return (
-        <div
-          className={cx(styles.arrowButton, { [styles.arrowButtonOver]: this.props.over })}
-        >
-          <Icon
-            icon="arrow-left-drop-circle-outline"
-            size={this.props.size}
-            color={this.props.color}
-            className={cx(styles.imgBottom, { [styles.imgOver]: this.props.over })}
-          />
-          <Icon
-            icon="arrow-left-drop-circle"
-            size={this.props.size}
-            color={this.props.color}
-            className={cx(styles.imgTop, { [styles.imgOver]: this.props.over })}
-          />
-        </div>
-      );
-    }
-    return (
+function Arrow({
```

Comment:
Nós temos usado const em vez de function

Reply:
@juliabpp thank you feedback. I will change it using const.